src/views/home/PlayerHome/PlayerHome.vue

```diff
@@ -122,38 +175,84 @@
     get nextPuzzleID() {
       return this.eternaEssentialsAchievements[0].current_puzzle;
     }
+
+    data() {
+      return {
+        calendarOptions: {
+          initialView: 'upcoming',
+          headerToolbar: false,
+          plugins: [
+            dayGridPlugin,
+            interactionPlugin,
+            googleCalendarPlugin,
+            listPlugin,
+          ],
+          views: {
+            upcoming: {
+              type: 'list',
+              duration: { days: 3 },
+              listDayAltFormat: 'dddd',
+            },
+          },
+          height: 'auto',
+          eventClick: this.handleEventClick,
+          googleCalendarApiKey: process.env.VUE_APP_GOOGLE_API_ID,
+          events: {
+            googleCalendarId: process.env.VUE_APP_GOOGLE_CALENDAR_ID,
+          },
+        },
+      };
+    }
+
+    handleEventClick(info: {
```

Comment:
Can you walk me through what this is for? It looks like it's FullCal related?

Reply:
This function overrides the default behavior in full calendar so when you click an event it opens google calendar in a new tab instead of using the current one.